app/controllers/temp/solutions_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Temp
```

Comment:
@iHiD This is the temporary controller.

Reply:
Implemented here: https://github.com/exercism/website/pull/564